### ISSUE TITLE
SplitButton contains a typo on "aria-haspopup" attribute

### ISSUE THREAD

**Issue by Cue-Ai-NGinEar** [primefaces/primeng#14533]
### Describe the bug

In file [src/app/components/splitbutton/splitbutton.ts](https://github.com/primefaces/primeng/blob/398278dec77d3c3389aed1c5a1674838653d8256/src/app/components/splitbutton/splitbutton.ts#L57) the "aria-haspopup" attribute has a typo - implemented as "aria-aria-haspopup". Pretty basic and straightforward to fix.

### Environment

Bug in source code. Can also be seen at primeng.org/splitbutton by inspecting the component. 

### Angular version

N/A

### PrimeNG version

17.0.5

### Build / Runtime

TypeScript

### Language

TypeScript

### Node version (for AoT issues node --version)

N/A

### Steps to reproduce the behavior

Simply inspect the split button component. Axe and WAVE scans highlight the invalid aria attribute on this component. 

### Expected behavior

Aria attribute does not have a typo that means it is flagged by accessibility tests.
